### ISSUE TITLE
Needs branding!

### ISSUE THREAD

**Issue by ericelliott** [ericelliott/essential-javascript-links#1]
* [ ] color scheme
* [ ] logo
* [ ] typography

**Comment by tristandenyer** [comment]
Any logos or branding that you really like? Node? HTML5? Just looking for some guidance on style.

**Comment by ericelliott** [comment]
Well, I have a logo for my courses... Maybe you can use that for inspiration?

![eric-elliott-logo](https://cloud.githubusercontent.com/assets/364727/6088270/d5809bcc-ae11-11e4-8019-fbb6d12aa436.png)

**Comment by anthonybrown** [comment]
How can I help, do I need to do a pull or can I just create a github page with this repo?
I work with git all the time, just never did a pull request

**Comment by ericelliott** [comment]
For now, I'd just drop a logo image into this comment thread. I think we should look at several ideas before we decide on an official logo. =)

**Comment by AllThingsSmitty** [comment]
Not very original, but it's a start...

![es-js](https://cloud.githubusercontent.com/assets/1682367/6141200/26e9f754-b170-11e4-9f4e-1b39f8533ff5.gif)

**Comment by tristandenyer** [comment]
I did a quick draft of one. Beginning to think the list/E icon is a bit thick. Looking for some feedback.
![essential-javascript-logo](https://cloud.githubusercontent.com/assets/4358288/6142446/75c45748-b16b-11e4-9246-bceb559dcf33.gif)
Will try and sketch up some of the other ideas I had tomorrow.

**Comment by ericelliott** [comment]
@AllThingsSmitty 

That might actually be useful in some context, but I'm not sure it would work as a logo glyph. =)

@tristandenyer I like where you're going with this. Can't wait to see some iterations...

**Comment by tristandenyer** [comment]
Ok, been buried in work. I'll post a couple sketches to see if I should start designing these too. Super rough ones done on the train, but you get the idea.
![ejs-1](https://cloud.githubusercontent.com/assets/4358288/6191995/b687a70a-b362-11e4-9247-cc9505e3767f.jpg)
![ejs-2](https://cloud.githubusercontent.com/assets/4358288/6191996/b99cf828-b362-11e4-8216-3218389ededd.jpg)

**Comment by tristandenyer** [comment]
I made the todo list icon a bit thinner, then did a couple variants on that. Let me know what you think. I'm open to all feedback.
![screen shot 2015-02-15 at 10 23 52 am](https://cloud.githubusercontent.com/assets/4358288/6204350/15ec80da-b4fd-11e4-99a2-4fcfaec6ceb9.png)

**Comment by ericelliott** [comment]
Nice work! =)

**Comment by AllThingsSmitty** [comment]
@tristandenyer that's good stuff.

**Comment by Cmdv** [comment]
The bottom one of the three looks nice to me @tristandenyer better alignment with the written word.
I've made something but I'm really not sure what I've done but thought I'd share it anyways, think I was going for something less formal I guess maybe someone else can take it to a better place!?!

![screen shot 2015-02-17 at 23 27 13](https://cloud.githubusercontent.com/assets/3036557/6239520/875b2f90-b6fc-11e4-9520-ffa7230fc0b1.png)

**Comment by ericelliott** [comment]
@Cmdv Haha, I love the smiles. =)

**Comment by Cmdv** [comment]
haha a little more friendly but the font I used is a bit rubbish, I'll ask someone who designs if they can give a hand, is the wording a must or EJS a good way to go?

**Comment by phoenixstormcrow** [comment]
The problem with any variant of "ejs" is that that name is taken already. Really like where @tristandenyer is going with this.

**Comment by ericelliott** [comment]
I'm not judging at this stage. I say get creative and throw ideas up. I am not concerned that there is an EJS package on npm. Every three letter acronym ever dreamed up is taken already by somebody. That's OK. =)

Thanks for the contributions so far!

**Comment by tristandenyer** [comment]
A quick mashup.
![screen shot 2015-02-20 at 9 46 06 am](https://cloud.githubusercontent.com/assets/4358288/6306124/89648350-b8e5-11e4-8013-4f499c3880c8.png)

**Comment by AllThingsSmitty** [comment]
@tristandenyer I dig that!

**Comment by ericelliott** [comment]
That smiley logo needs a theme song.

**Comment by Cmdv** [comment]
![screen shot 2015-02-20 at 22 01 57](https://cloud.githubusercontent.com/assets/3036557/6310174/25c41dbc-b94c-11e4-8629-a71eb46be8c0.png)

**Comment by tristandenyer** [comment]
I'm liking both of these, @Cmdv.

**Comment by Cmdv** [comment]
cheer @tristandenyer I merged the idea of the E and put it into the list, maybe that needs a bit of work, it's all Illustrator so can pass the file round if need be =)

**Comment by ericelliott** [comment]
The clipboard one looks great. =)

**Comment by tristandenyer** [comment]
I put @Cmdv's clipboard logo through one of the most intensive unit tests I know of and I like it!

![ejs-laptop-lid](https://cloud.githubusercontent.com/assets/4358288/6310551/e12c3a20-b90c-11e4-9ac9-78f756b2d6e2.jpg)

**Comment by RoselPadilla** [comment]
![essentialjs](https://cloud.githubusercontent.com/assets/11068746/6311451/f8e29776-b930-11e4-8c1e-1564a61e5d40.png)

**Comment by ericelliott** [comment]
@tristandenyer functional testing for logos! Can we automate that? ;)

@RoselPadilla Nice work. I like it! =)

**Comment by Cmdv** [comment]
@tristiandenyer haha that's some heavy unit testing! still think the E needs slight work to make it stand out a little more maybe thinker?!? I'll have another tinker later

**Comment by phoenixstormcrow** [comment]
@ericelliott well put. My own contribution is only a conceptual one: the branding idea that won't leave me alone is that there ought to be some way to incorporate the use, or abuse, of the custom element syntax, viz:

```<script is="essential-js"></script>```

This looks pretty good to me displayed large in a mono font. Not really suitable for a logo, but I like the concept; maybe someone more designy than I can do something with it.

**Comment by AllThingsSmitty** [comment]
@Cmdv really dig that clipboard and @tristandenyer's sticker proof.

**Comment by beejaymay** [comment]
Inspiration struck.

![essentialjs](https://cloud.githubusercontent.com/assets/5924469/6598568/8ddb6628-c7d2-11e4-87a8-3f8c1a0447f8.png)

**Comment by beejaymay** [comment]
See what I did there?

**Comment by ericelliott** [comment]
Cool! =)

**Comment by radiovisual** [comment]
**Here is my idea:** *using the javascript syntax as symmetrical design elements.* Because the syntax is essential, and javascript! =)

![essential-js-logo-ideas](https://cloud.githubusercontent.com/assets/5614571/6642377/13874c90-c99c-11e4-9320-28ebc990d10e.jpg)

**Comment by AllThingsSmitty** [comment]
@radiovisual that's slick!

**Comment by radiovisual** [comment]
Here is a variation from my first theme, where the elements are more inline, and balanced. We can mess with the colors if you like the direction. 

![essential-js-logo-ideas-02](https://cloud.githubusercontent.com/assets/5614571/6642715/b837338e-c99e-11e4-8b97-3a7cec3d8fda.jpg)

**Comment by radiovisual** [comment]
Another variation on my original theme: dropped "(js)" to a new line, and removed the semicolon. 

![essential-js-logo-ideas-03](https://cloud.githubusercontent.com/assets/5614571/6642887/0fe9d57c-c9a0-11e4-8057-1a76f7bca445.jpg)

**Comment by radiovisual** [comment]
and finally, with a little color blocking:

![essential-js-logo-ideas-05](https://cloud.githubusercontent.com/assets/5614571/6643966/5fbcd452-c9a8-11e4-8e54-4819e5e5d4a4.jpg)

Let me know if I need to stop posting like a madman. I have no idea if I am violating some sort of etiquette here.

**Comment by radiovisual** [comment]
I consolidated all my drafts thus far into my original post, since I was getting carried away in photoshop...

**Comment by radiovisual** [comment]
A few new ideas: clean, simple typography:
 
![essential-js-logo-ideas-09](https://cloud.githubusercontent.com/assets/5614571/6651662/e73b624e-ca44-11e4-9f00-b5c2778ed6c7.jpg)

**Comment by ericelliott** [comment]
So many awesome ideas. =)

**Comment by stoeffel** [comment]
:+1: for the clipboard logo from @Cmdv

**Comment by ndedic** [comment]
+1 for the clipboard logo

**Comment by radiovisual** [comment]
Even though I have submitted a bunch of ideas myself, I am rather fond of @Cmdv 's  clipboard logo as well, but it was @tristandenyer 's "unit test" that sold it for me.

**Comment by ericelliott** [comment]
> it was @tristandenyer 's "unit test" that sold it for me.

Yeah, I almost thing that's a little unfair (but really cool). Some of these other options might do really well in that test, too! =)

**Comment by AllThingsSmitty** [comment]
:+1: clipboard logo!

**Comment by tristandenyer** [comment]
Super-quick mockup of some of the designs above. Lots of good stuff here.
![mac-lid-sticker-test](https://cloud.githubusercontent.com/assets/4358288/6659908/406a21e0-cb4c-11e4-9bcb-1df6d71d7c6d.jpg)

**Comment by rsperberg** [comment]
I like the look of @radiovisual's tall JS with the "essential" bar across it, and the play with angle brackets, square brackets and parentheses, but @Cmdv's clipboard is so distinctive it blows the others away, imo.

The only problem I have with it is that the size of the "js" seems to be off somehow. Perhaps if caps were used (particularly the J with a hook that @radiovisual uses in some logos) the right side would hold up better?

**Comment by ericelliott** [comment]
Thanks @tristandenyer!

Are these unit tests or functional tests? ;)

**Comment by tristandenyer** [comment]
I could go either way, but functional is probably more correct. :) It's highly scientific, that's for sure.

**Comment by ericelliott** [comment]
:+1:

**Comment by tinkertrain** [comment]
Hi all, here are some doodles I made thinking about this, on one side I thought about departing from the literal “essential javascript list” with a character of some kind; the others (letter cube and water-plant) have more literal connection to “Essential JavaScript”

![dibujillos](https://cloud.githubusercontent.com/assets/1414086/6693624/b66ab436-ccaa-11e4-9377-120c9ac6745e.jpg)

**Comment by ericelliott** [comment]
haha! These are awesome!

**Comment by radiovisual** [comment]
@tinkertrain . Really awesome work. I love the water with the plant, since I think it fits the theme of learning and growing, and also because water is *essential*.  I also love the character sketches, particularly the animal top right. +1

**Comment by tinkertrain** [comment]
@radiovisual That is exactly what I was thinking with the water drop :smile: 

Here's some progress, still plenty of work left to do:
![essential plant sketch 2015-03-18 13-32-58](https://cloud.githubusercontent.com/assets/1414086/6715273/c7075b68-cd73-11e4-8840-d99c3aace52d.png)

**Comment by hoichi** [comment]
I'm kinda late to the party and it's not quite a logo idea but how about a [site] name like JSsential? It doesn't seem to be taken yet. All that Google gave me looks like typos.

**@tinkertrain** maybe a drop flowing _out_ of "JS"? I mean, we're talkng about the essence _of_ JavaScript ;)

**Comment by apnerve** [comment]
@radiovisual I really dig that design with color blocking!

**Comment by radiovisual** [comment]
thanks @apnerve ! :smile:

**Comment by pertrai1** [comment]
I am all for @radiovisual tall JS logo.

**Comment by hoichi** [comment]
I, on my part, think that all those @radiovisual’s `e{js}` variants feel way more JS-y than the tall logo, as sweet as it is per se.

**Comment by TimbreDesign** [comment]
Hey there, maybe a little late to the party (fashionably I hope).. I've got some ideas to swing by yall:
![screenshot 2015-06-27 14 22 47](https://cloud.githubusercontent.com/assets/13074081/8390773/54fc4dac-1cd9-11e5-9c60-4dd3840edcfa.png)
![screenshot 2015-06-27 14 22 12](https://cloud.githubusercontent.com/assets/13074081/8390775/56356884-1cd9-11e5-90e0-c5c5222cd6f3.png)
![screenshot 2015-06-27 14 31 06](https://cloud.githubusercontent.com/assets/13074081/8390782/73011d46-1cd9-11e5-97ca-c9b889e758b2.png)

**Comment by cdvillard** [comment]
This kind of died down six months ago, but looking at this focus on Essential JavaScript, I'd like to point out the initials are already pretty popular (i.e.: Eloquent JavaScript). For SEO purposes, I would suggest something along the lines of JS Essentials or ECMA Essentials to separate it from other similarly acronymed titles.

**Comment by ericelliott** [comment]
I'm not too worried about the SEO value of the acronym. You'd have the same problem with any name, including "JSE".

**Comment by timsvoice** [comment]
Hi all, some possible logo ideas. One in response to the above, and a couple iterations of another idea. Thoughts welcome.

![clipboard](https://cloud.githubusercontent.com/assets/2458150/19168422/17790072-8bde-11e6-8283-b417e1392cdd.png)

Beaker with "essential" underneath 

![beaker_under](https://cloud.githubusercontent.com/assets/2458150/19168424/177e8c18-8bde-11e6-8555-4c515a46d4d1.png)

Beaker with "essential" inside

![beaker_inside](https://cloud.githubusercontent.com/assets/2458150/19168421/177861f8-8bde-11e6-8563-7e3625300434.png)

Beaker with no "essential"

![beaker_simplest](https://cloud.githubusercontent.com/assets/2458150/19168420/1777f7b8-8bde-11e6-9faf-c203d7658fbf.png)

Beaker with bubbles for fun!

![beaker_bubbles](https://cloud.githubusercontent.com/assets/2458150/19168423/1779956e-8bde-11e6-8c36-aa815f4b6e96.png)

**Comment by ericelliott** [comment]
Hmm.. the beaker with bubbles is pretty neat. Also, the lines remind me a bit of my logo. See [EricElliottJS.com](https://ericelliottjs.com) to see what I mean.

**Comment by timsvoice** [comment]
@ericelliott yeah, I see what you mean. 

Had a moment to work on a simple layout concept today. WIP screenshot below
Obviously the branding will change to reflect the consensus.

![desktop hd_screenshot](https://cloud.githubusercontent.com/assets/2458150/19206375/83dc1f32-8cb6-11e6-8a63-8cbe3c4fbc05.png)

**Comment by timsvoice** [comment]
With the beaker bubbles logo and simplified top-bar branding

![desktop hd_screenshot_bubbles](https://cloud.githubusercontent.com/assets/2458150/19207445/02c0ac20-8cbf-11e6-9834-dd972c9aeb16.png)

**Comment by timsvoice** [comment]
Oh, also had these "essential oil" experiments   ¯\_(ツ)_/¯ 

<img width="183" alt="screen shot 2016-10-07 at 7 53 06 pm" src="https://cloud.githubusercontent.com/assets/2458150/19208433/be51f9aa-8cc7-11e6-850c-bf074697539e.png">

**Comment by apnerve** [comment]
Loved the bubbles one.

**Comment by BenGitsCode** [comment]
I agree with @apnerve I love the bubbles. The essential jars are cool but just make me think of peanut butter—which is an unfair bias.

**Comment by timsvoice** [comment]
@BenGitsCode, ha! you're right! I didn't see the peanut butter...

Some refinement on the beaker with bubbles version

![beaker_inside_bubbles](https://cloud.githubusercontent.com/assets/2458150/19225142/98fafdd0-8e63-11e6-83f3-14262358e7aa.png)

**Comment by ericelliott** [comment]
@timsvoice I think I liked the previous beaker better. In this one, the prominent Eric Elliott logo looks like it's trying to compete with the "Essential JS" branding, which create's a weaker focus and weaker anchor.

That said, I've never been a big fan of the JavaScript yellow. Maybe try blue inspired by the [EricElliottJS branding](https://github.com/ericelliott/essential-javascript-links/issues/1#issuecomment-73321782)

**Comment by timsvoice** [comment]
@ericelliott good point re: the competing elements. 

Here are three responses with varying degrees of 3D and shading ranging from totally flat

![beaker_inside_bubbles black blue](https://cloud.githubusercontent.com/assets/2458150/19255401/88e5fc88-8f2b-11e6-8908-87636ec2e22f.png)

to having some depth with the beaker contents

![beaker_inside_bubbles black blue 3d-ish](https://cloud.githubusercontent.com/assets/2458150/19255374/40d4f8fe-8f2b-11e6-8e27-b85d60dfca5a.png)

to full depth with shading on the bubbles

![beaker_inside_bubbles black blue 3d](https://cloud.githubusercontent.com/assets/2458150/19255378/457bf36c-8f2b-11e6-8506-48ee5901bcc2.png)

**Comment by ericelliott** [comment]
I like the middle option best. Can the lines be light grey instead of black?

**Comment by timsvoice** [comment]
Yeah, I think that's the best option, too. Though it breaks a few perspective rules, I think it works.

Here with the EE logo in grey:

![beaker_inside_bubbles light gray blue 3d-ish](https://cloud.githubusercontent.com/assets/2458150/19402651/2e8ebe7a-9230-11e6-8524-2a09e8718cf1.png)

**Comment by jdsninja** [comment]
Hi Guys,
Here's something new made by [Aïcha Barry](http://aichabarry.com) (She is a visual designer with no github account :).
What do you think?

![jsessentialbyaichabarry](https://cloud.githubusercontent.com/assets/666144/19493282/064304ba-9548-11e6-9ca4-5d4e9e82b53c.png)

**Comment by jdsninja** [comment]
Almost done!

![screen shot 2016-10-18 at 11 30 53 pm](https://cloud.githubusercontent.com/assets/666144/19504827/719d8efe-958b-11e6-94ca-bbe9505adc82.png)

**Comment by jdsninja** [comment]
@ericelliott check this out :)
![essentieljs_presentation](https://cloud.githubusercontent.com/assets/666144/19737159/5814a1a2-9b80-11e6-9e9c-b249681a379d.png)
